### PR TITLE
ci(): publish @next tag on PR merge

### PR DESCRIPTION
**Pull request by ShaMan123**
#7256

### CODE REVIEW

File: .github/workflows/npmpublish-next.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    branches: [ master ]
+
+jobs:
+  publish-npm:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
```
Comment (asturur) [inline .github/workflows/npmpublish-next.yml:14]: let's bump this to 16 since is the current LTS.

Comment (asturur) [reply .github/workflows/npmpublish-next.yml:14]: (we will do it for the other scripts too later)

**Comment by ShaMan123** [comment]
Did it work?

**Comment by ShaMan123** [comment]
This try didn't work right?

**Comment by asturur** [comment]
no i don't think is working at all.
Not sure if is permission or else, i did not check anymore

**Comment by ShaMan123** [comment]
up

**Pull request merged**